it/test/connectors/CitizenDetailsConnectorISpec.scala

```diff
@@ -67,7 +67,7 @@ class CitizenDetailsConnectorISpec extends AnyWordSpec with IntegrationSpecBase
 
           val result = connector.getPersonDetails(nino).futureValue
 
-          result mustBe Left(ErrorResponse(500, "Call to citizen details failed"))
+          result mustBe Left(ErrorResponse(500, "{}"))
           WiremockHelper.verifyGet(s"/citizen-details/${nino.value}/designatory-details")
```

Comment:
Could have a little test for the recover so when if the get fails it goes into the recover case